lib/live_debugger/utils/logger.ex

```diff
@@ -0,0 +1,28 @@
+defmodule LiveDebugger.Utils.Logger do
+  @moduledoc """
+  This module extends the Logger module with additional information added to messages.
+  """
+
+  require Logger
+
+  def info(message) when is_binary(message), do: Logger.info(message)
+  def warning(message) when is_binary(message), do: Logger.warning(message)
+
+  @spec error(message :: String.t(), report_message? :: boolean()) :: :ok
+  def error(message, report_message? \\ true) when is_binary(message) do
+    error_message =
```

Comment:
Please check if this works correctly in `prod` (when LiveDebugger is added as dep). You can try using `raise` somewhere in the code to trigger error